### DISCUSSION TITLE
How to map default `:w` file management command from `save` to `save without formatting`?

### DISCUSSION THREAD

**Discussion by zhuoqun-chen** [vscode-neovim/vscode-neovim#2009] [Q&A] [answered]
As the title suggested, currently I'd like to pop up the cmdline in normal mode in vscode-neovim, type in `:w` and the VSCode command `File: Save without Formatting` be triggered instead of the default `File: Save`.

Is there a way to achieve this?

**Comment by ollien** [comment, marked as answer]
I don't think there is. This is hardcoded to `saveAs` and `save`

https://github.com/vscode-neovim/vscode-neovim/blob/833b38ca75ace08150b5c76edbd8b05ed711d882/vim/vscode-file-commands.vim#L34

**Reply by justinmk** [reply]
Right. This is more of a (known/existing) feature request for Nvim/Vim. vscode-neovim provides some aliases for `:write` and similar, but it's a hack. 

Discussed in:

- https://github.com/vscode-neovim/vscode-neovim/issues/136
- https://github.com/vscode-neovim/vscode-neovim/issues/1260

**Reply by zhuoqun-chen** [reply]
Thank you @ollien and @justinmk for the help and information, however, this is a little bit too complicated for my use case to hack it. I think I can set other keybindings.

Wish you a good day!
